apis/nucleus/src/index.js

```diff
@@ -67,33 +47,69 @@ const DEFAULT_SNAPSHOT_CONFIG = /** @lends SnapshotConfiguration */ {
 
 /**
  * @interface Configuration
+ * @property {LoadType=} load Fallback load function for missing types
```

Comment:
how does this use interact with getRegisteredTypes? (and should there be anything in the docs about it?)

Reply:
hmm, have to think about that. I think I added that to solve something with charts in tooltips, but not sure. For coming projects we will need it however, but in an expanded sense. So might be that we need another config function to make it fully compatible.